Makefile

```diff
@@ -1,35 +1,47 @@
-.PHONY: all
-all: check
+GO111MODULE = on
+export GO111MODULE
+GOFLAGS += -mod=vendor
+export GOFLAGS
+GOPROXY ?=
+export GOPROXY
 
 NO_DOCKER ?= 0
 ifeq ($(NO_DOCKER), 1)
   DOCKER_CMD =
   IMAGE_BUILD_CMD = imagebuilder
-  CGO_ENABLED = 1
 else
-  DOCKER_CMD := docker run --rm -v "$(PWD)":/go/src/github.com/openshift/cluster-api-actuator-pkg:Z -w /go/src/github.com/openshift/cluster-api-actuator-pkg openshift/origin-release:golang-1.12
+  DOCKER_CMD := docker run \
+	  --rm \
+	  -v "$(PWD)":/go/src/github.com/openshift/cluster-api-actuator-pkg:Z \
+	  -w /go/src/github.com/openshift/cluster-api-actuator-pkg \
+	  -e "GO111MODULE=$(GO111MODULE)" \
+	  -e "GOFLAGS=$(GOFLAGS)" \
+	  -e "GOPROXY=$(GOPROXY)" \
+	  openshift/origin-release:golang-1.12
   IMAGE_BUILD_CMD = docker build
 endif
 
-.PHONY: depend
-depend:
-	dep version || go get -u github.com/golang/dep/cmd/dep
-	dep ensure
+.PHONY: all
+all: check
 
-.PHONY: depend-update
-depend-update:
-	dep ensure -update
+.PHONY: vendor
+vendor:
```

Comment:
This appears to be idempotent; I can run it on this patch and get no changes.  I can rm -rf vendor/ and get no changes.  I'm unsure how it will be have across multiple projects being updated and then run this.  But appears to work for now.